app/views/bets/index.html.erb

```diff
@@ -5,5 +5,9 @@
       <h4>This page will display all the existing bets (25/page) and allow the user to use custom search criteria.</h4>
     </div>
   </div>
-  <%= render 'bets/display_bets' %>
+  <div class="d-flex flex-column">
+    <div class="p-2">
+      <%= render 'bets/display_bets', bets: @bets %>
```

Comment:
Nice! I really like that you pass the instance variable into the partial. This makes the partial way easier to reuse elsewhere.

For reference, the @bets instance variable is still available within the partial, but its bad practice to use instance variables in partials.

Just as as reminder that rails will start looking for partials in the current view directory. So in this case 
`<%= render 'display_bets', bets: @bets %>` would also have worked, since the index view and display bets are in the same view directory.


Another option available to you here is that Rails can infer which view to render from AR models. Making it possible to write

`<%= render @bets %>` 

This will look for a partial called `/bets/_bet.html.erb` and it will automatically loop over ever item in the @bets collection and render 1 partial for each bet. 

http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections